build.sbt

```diff
@@ -75,6 +75,22 @@ lazy val mimaSettings = Seq(
   mimaBinaryIssueFilters ++= Seq()
 )
 
+val beamSDKIO =
+  Seq(
+    "org.apache.beam" % "beam-sdks-java-io-google-cloud-platform" % beamVersion excludeAll(
+        ExclusionRule("com.google.cloud", "google-cloud-spanner"),
+        ExclusionRule("com.google.cloud", "google-cloud-core"),
+        ExclusionRule("com.google.api.grpc", "proto-google-cloud-spanner-admin-database-v1"),
+        ExclusionRule("com.google.api.grpc", "proto-google-common-protos")
+      ),
+    "io.grpc"  % "grpc-core" % "1.7.0",
```

Comment:
Do we have to pin GPRC version? Can we exclude bad ones and let sbt pick the latest instead? Or least have a `val grpcVersion` so it's more manageable?

Reply:
If we don't version `1.2.0` is selected. Most dependencies require `1.7.0`